conformity/fields/geo.py

```diff
@@ -1,8 +1,21 @@
 from __future__ import absolute_import, unicode_literals
 
 import attr
+import pycountry
```

Comment:
As mentioned above, this creates a hard dependency on `pycountry`. We need to make this a soft dependency. So we have two options:
1) Put all of these additions in a separate module; or,
2) Do this:

```
try:
    import pycountry

    _countries_a2 = [c.alpha_2 for c in pycountry.countries]

    class CountryCodeField(Constant):
        ... [the rest] ...
except ImportError:
    pass
```